src/comparator.html

```diff
@@ -327,7 +327,64 @@ <h2 class="absolute top-0 left-1 pa1 sans-serif f4 bg-white o-60">
               console.error(error);
               applyUserStyleFromStorage();
             });
-        } else {
+        } // Github commit
+        if (params.has("github")) {
+          const github = params.get("github");
+          const ref = params.get("ref") !== 0 ? params.get("ref") : "";
+          const url = `https://api.github.com/repos/${github}/contents/style.json${
+            ref ? "?ref=" + ref : ""
+          }`;
+
+          fetch(url, {
+            headers: {
+              Accept: "application/vnd.github.v3.raw",
+            },
```

Comment:
Do we anticipate that these repos will be public? If so, we can omit credentials to github. In the very unlikely event that this ends up making a request to another domain, we won't accidentally include credentials when we don't need to.

If we need to support private repos, then ignore this suggestion, as you'd need to supply github with credentials (I assume)

```suggestion
            },
            credentials: 'omit',
```

Reply:
yep, they all should be public, thanks for the suggestion :heart: